lms_latest.json

```diff
@@ -0,0 +1,54 @@
+{
+	"Logitech Media Server - Latest Stable": {
+		"description": "Logitech Media Server (aka Squeezebox, slimserver, etc.). Latest stable version. For future updates see more information icon after first installation<p>Uses custom docker image: <a href='https://hub.docker.com/r/doliana/logitech-media-server' target='_blank'>https://hub.docker.com/r/doliana/logitech-media-server</a>. Available for amd64, arm/v7 &arm64 architecture.",
+		"more_info": "<h4>Update to newer version</h4><p>After first installation: to update to a newer stable version, either uninstall/reinstall the Rockon or use the Watchtower Rockon to manage the update of the underlying Media Server image, as it does not automatically update itself during a stop/restart of the Rockon like some other Rockons do. Don't use the update option within the LMS!</p><p>The LMS configuration files will remain in place on the volumes that were specified during the initial installation</p>",
```

Comment:
Similar to the previous LMS Rock-on you worked on, maybe we should mention that the webUI port MUST be set to 9000. You had the following mention in the previous LMS Rock-on:
```
<p>Due to some current <a href='https://github.com/rockstor/rockstor-core/issues/2209' target='_blank'>limitations in rock-on networking</a>, the web UI port of this rock-on must be set as 9000 and will thus prevent the installation of other rock-ons needing this port. Alternatively, the web UI can be accessed manually at http://rockstor-ip:9000</p>
```